src/server/debug/server/server.go

```diff
@@ -759,23 +859,45 @@ func collectGoInfo(tw *tar.Writer, prefix ...string) error {
 		}
 		fmt.Fprintf(w, "GOOS: %v\nGOARCH: %v\nGOMAXPROCS: %v\nNumCPU: %v\n", runtime.GOOS, runtime.GOARCH, runtime.GOMAXPROCS(0), runtime.NumCPU())
 		return nil
-	}, prefix...)
+	})
 }
 
-func (s *debugServer) collectPachdVersion(ctx context.Context, tw *tar.Writer, pachClient *client.APIClient, prefix ...string) error {
-	return collectDebugFile(tw, "version", "txt", func(w io.Writer) (retErr error) {
-		defer log.Span(ctx, "collectPachdVersion")(log.Errorp(&retErr))
+func (s *debugServer) collectPachdVersion(ctx context.Context, dfs DumpFS, pachClient *client.APIClient) error {
+	return dfs.Write("/pachd/version.txt", func(w io.Writer) error {
```

Comment:
Someday this probably wants to ask each component (other pachd replicas, pachw, workers) their version, so we can detect faulty upgrades.